client/src/components/Navigation.js

```diff
@@ -7,56 +7,78 @@ import { ReactComponent as WalletIcon } from '../image/wallet.svg';
 export default function Navigation() {
   return (
     <NavigationWrapper>
-      <NavBar>
-        <NavLink exact to="/" className="navButton">
-          <NavigationIcon>
-            <NewsIcon />
-          </NavigationIcon>
-        </NavLink>
-
-        <NavLink to="/bookmarks" className="navButton">
-          <NavigationIcon>
-            <BitcoinIcon />
-          </NavigationIcon>
-        </NavLink>
-
-        <NavLink to="/wallet" className="navButton">
-          <NavigationIcon>
-            <WalletIcon />
-          </NavigationIcon>
-        </NavLink>
-      </NavBar>
+      <NavLinkStyled to="/news" className="navButton">
+        <IconNameWrapper>
+          <NewsIcon />
+          <IconName>News</IconName>
+        </IconNameWrapper>
+      </NavLinkStyled>
+
+      <NavLinkStyled to="/bookmarks" className="navButton">
+        <IconNameWrapper>
+          <BitcoinIcon />
+          <IconName>Bookmarks</IconName>
+        </IconNameWrapper>
+      </NavLinkStyled>
+
+      <NavLinkStyled to="/wallet" className="navButton">
+        <IconNameWrapper>
+          <WalletIcon />
+          <IconName>Wallet</IconName>
+        </IconNameWrapper>
+      </NavLinkStyled>
     </NavigationWrapper>
   );
 }
 
-const NavigationIcon = styled.span`
-  svg {
-    width: 2rem;
-    height: 2rem;
-    fill: white;
-  }
-`;
-
 const NavigationWrapper = styled.div`
-  display: flex;
-  justify-content: center;
-`;
+  background: repeating-linear-gradient(
```

Comment:
please consider sorting the css styles in alphabetical order (:

Reply:
fixed :)